doc/news/changes/minor/20200325RezaRastak

```diff
@@ -0,0 +1,4 @@
+Added: New methods added to the DiscreteTime class including get_step_number()
+, get_previous_time(), is_at_start(), is_at_end(), get_previous_step_size().
```

Comment:
put the comma at the end of the previous line

Reply:
Fixed